projects/conductor/package.json

```diff
@@ -10,7 +10,7 @@
   "description": "",
   "devDependencies": {
     "@fingerprintjs/fingerprintjs-pro": "^3.11.6",
-    "@fingerprintjs/fingerprintjs-pro-server-api": "^6.9.0",
+    "@fingerprintjs/fingerprintjs-pro-server-api": "6.10.0-test.1",
```

Comment:
You forgot this one

Reply:
Please see this commit https://github.com/fingerprintjs/dx-team-orchestra/pull/55/commits/01eb246bfecd64cf2b335490a23eddf31a7d3a15